cjwilburn/dank-memes
# Explain to my mom meme generator

# Description
Addresses [issue 56](https://github.com/cjwilburn/dank-memes/issues/56) - Explain to my mom meme.

- Generates text and api call to generate a custom explain to my mom meme
- Adds the ability to read from a `.env` for credentials
- Adds `.env` to `.gitignore`
- Adds tests using Ava 

## Steps to Test
1. Create a `.env` and add your credentials (If you don't have an account to generate memes from the imgflip api, you can [sign up here](https://api.imgflip.com/)). This generator is expecting values for `IMGFLIP_USER` and`IMGFLIP_PASSWORD`
2. `npm install`
3. `node explain-to-mom.js`
4. Follow the prompts
5. Run tests with `npm run test`. Tests should pass. 


## Screenshots
![image](https://user-images.githubusercontent.com/2473702/67580093-94c97b80-f6fa-11e9-92f4-25906c159510.png)


## Checklist
